graphql/schema/types/config.graphql

```diff
@@ -9,7 +9,6 @@ type SpruceConfig {
   containerPools: ContainerPoolsConfig
   githubOrgs: [String!]!
   jira: JiraConfig
-  keys: [SSHKey!]!
```

Comment:
Same comment as above this is not removable until step 5 from above. The UI should no longer rely on this at that point.